### DISCUSSION TITLE
Mixed node pools support

### DISCUSSION THREAD

**Discussion by SludgeX** [linkerd/linkerd2#7342] [Q&A] [answered]
Hi, I'm trying to test Linkerd on a new AKS cluster, so far the install procedure has been very smooth but I've ran into an issue. The sidecar injection works on a service scheduled for a Linux node but fails when for a service targeting a windows node. 

Is this something i missed in the documentation? Are windows nodes not supported by Linkerd?

Thanks in advance.

**Comment by alpeb** [comment, marked as answer]
That's correct, the linkerd proxy sidecar can't be installed on windows nodes because it relies on `iptables` to reroute connections to/from the pod, which isn't available in windows nodes.

**Reply by SludgeX** [reply]
Ah, that is unfortunate, but thank you for the quick answer.
I know that this is no longer related to Linkerd so feel free to disregard this follow up question, but do you know of service meshes that do support windows nodes?

**Reply by alpeb** [reply]
Sorry, I'm afraid I don't know of any :man_shrugging:

**Reply by SludgeX** [reply]
Ok, thank you for your help.
